csm_web/frontend/src/components/section/SpacetimeEditModal.tsx

```diff
@@ -10,6 +10,7 @@ interface SpacetimeEditModalProps {
   closeModal: () => void;
   defaultSpacetime: Spacetime;
   reloadSection: () => void;
+  overrideEdit: boolean;
```

Comment:
[nit] imo, `isEditingOverride` or `editingOverride` seems more intuitively a boolean; this is incredibly tiny, so feel free to ignore if you feel differently.